### PR TITLE
SORQA-217 - SORDEV-8048 fix and split

### PR DESCRIPTION
**Pull request by pk-sgent**
SORDEV-8048 - Test fixed and splited 

image

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java [hunk 1]
```diff
@@ -507,6 +508,22 @@ public CreateNewCaseSteps(
...
+          String getDisease = webDriverHelpers.getValueFromCombobox(DISEASE_COMBOBOX);
+          softly.assertEquals(disease, getDisease, "Diseases are not equal");
+          webDriverHelpers.clickOnWebElementBySelector(DISCARD_BUTTON_POPUP);
+        });
+
+    When(
+        "^I check if default disease value in the Line listing is set for ([^\"]*)$",
+        (String disease) -> {
+          String getDisease = webDriverHelpers.getValueFromCombobox(LINE_LISTING_DISEASE_COMBOBOX);
+          softly.assertEquals(disease, getDisease, "Diseases are not equal");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java:524]: you need to add also assert all

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java [hunk 2]
```diff
@@ -153,6 +161,14 @@ public CreateNewContactSteps(
           webDriverHelpers.clickOnWebElementBySelector(SOURCE_CASE_WINDOW_CONFIRM_BUTTON);
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(10);
         });
+
+    When(
+        "^I check if default disease value for contacts in the Line listing is set for ([^\"]*)$",
+        (String disease) -> {
+          String getDisease = webDriverHelpers.getValueFromCombobox(DISEASE_COMBOBOX);
+          softly.assertEquals(disease, getDisease, "Diseases are not equal");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java:169]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java [hunk 3]
```diff
@@ -376,6 +376,13 @@ public CreateNewSampleSteps(
           softly.assertAll();
           webDriverHelpers.clickOnWebElementBySelector(CONFIRM_BUTTON);
         });
+
+    When(
+        "I check if default disease value for new Pathogen test is set for ([^\"]*)",
+        (String disease) -> {
+          String testedDisease = webDriverHelpers.getValueFromCombobox(TESTED_DISEASE_COMBOBOX);
+          softly.assertEquals(disease, testedDisease, "Diseases are not equal");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java:384]: same

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java (hunk 1 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java:524]: omg .. :D sorry

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**